src/reducers/PagesReducer.js

```diff
@@ -21,7 +21,7 @@ export const PagesReducer = (state = {}, action) => {
       const { pageObject } = params;
 
       const pageInitialiser = getPageInitialiser(routeName);
-      const pageInitialState = pageInitialiser(pageObject);
+      const pageInitialState = pageInitialiser ? pageInitialiser(pageObject) : {};
```

Comment:
This will conflict with [#1517 changes](https://github.com/openmsupply/mobile/pull/1517/files#diff-f4800056faa9ea0c03eda6581a4e7969R24) but that's fine, though this here assigns an empty object where as that PR assigns null in the falsy case.